databases/arangodb/Portfile

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           openssl 1.0
+
+github.setup        arangodb arangodb 3.8.4 v
+revision            0
+categories-append   databases math
+license             Apache-2
+maintainers         nomaintainer
+supported_archs     x86_64
```

Comment:
Can Arrango not compile on arm64?

Reply:
Officially, no: https://github.com/arangodb/arangodb/issues/15301. I've confirmed myself.